examples/on_gke/e2e/README.md

```diff
@@ -0,0 +1,33 @@
+# Forseti on GKE - End-to-End
+Follow this example if new infrastructure is desired for the deployment of Forseti
```

Comment:
Is the intent of this example to demonstrate migrating from a VM deployment to a GKE deployment?

Reply:
No.  The scenario here is that a user is creating a brand new deployment of Forseti.  The on_gke sub-module requires buckets, service accounts, etc. to be setup prior to its invocation.

Reply:
The example assumes that the user has no Forseti, wants to deploy Forseti for the first time, and likes the idea of deploying Forseti on GKE.

Yes, the GCE vm's are deployed as well in this scenario.